app/experimenter/nimbus-ui/src/components/FormOverview/index.test.tsx

```diff
@@ -269,11 +272,11 @@ describe("FormOverview", () => {
       projects: [
         {
           name: "Pocket",
-          id: 1,
+          id: "1",
```

Comment:
same question as before, changing the int to string and will that make any impact on the existing experiments

Reply:
Answered above